docs/user_docs/installation/install-with-kbcli/install-kbcli.md

```diff
@@ -34,13 +34,30 @@ You can install kbcli with `curl` or `brew`.
    curl -fsSL https://kubeblocks.io/installer/install_cli.sh | bash
    ```
 
+   If you want to install kbcli with a specified version, follow the steps below.
+
+   1. Check the available versions in [KubeBlocks Release](https://github.com/apecloud/kubeblocks/releases/).
+   2. Specify a version with `-s` and run the command below.
+
+      ```bash
+      curl -fsSL https://kubeblocks.io/installer/install_cli.sh | bash -s x.x.x
+      ```
+
+  :::note
+
+  By default, kbcli installs KubeBlocks with the same version. If you want to install KubeBlocks with a different version, ensure that the minor versions of kbcli and KubeBlocks match.
```

Comment:
for `kbcli`, by default (no version/tag specified), the latest release version will be installed.

Reply:
> for `kbcli`, by default (no version/tag specified), the latest release version will be installed.

Fixed. Plz review docs again. THX!